portal-api/src/v1/users/passport.js

```diff
@@ -54,13 +64,21 @@ const baseAuthenticationConfiguration = ({ name, passport, additionalValidation,
   );
 };
 
-const loginCompleteAuth = (passport) => {
-  const additionalValidation = (jwtPayload) => jwtPayload.loginStatus === LOGIN_STATUSES.VALID_2FA;
-  baseAuthenticationConfiguration({ name: 'login-complete', passport, additionalValidation });
+const loginCompleteAuth = (passport, userService) => {
+  const name = 'login-complete';
+  const additionalPayloadValidation = (jwtPayload) => jwtPayload.loginStatus === LOGIN_STATUSES.VALID_2FA;
+  const additionalUserValidation = (user, jwtPayload) => {
+    if (userService.isUserBlockedOrDisabled(user)) {
+      console.error("User with username %s is blocked or disabled for '%s' strategy", jwtPayload.username, name);
+      return true;
```

Comment:
Surprising -- Could we add in a VALIDATION_RESULT const that is either Passed or Failed